### PR TITLE
CI: Switch from Travis to GH Actions

### PR DESCRIPTION
**Pull request by dinhtungdu**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      with:
+        php-version: '7.2'
+        coverage: none
+        tools: composer:v1
+
+    - name: Install dependencies
+      run: composer install
+
+    - name: Setup WP Snapshots
+      run: ./vendor/bin/wpsnapshots configure 10up --aws_key=${{ secrets.AWS_ACCESS_KEY_ID }} --aws_secret=${{ secrets.AWS_SECRET_ACCESS_KEY }} --user_name=wp-acceptance --user_email=travis@10up.com
```
Comment (jeffpaul) [inline .github/workflows/test.yml]: Is this email here still the correct one to use?

Comment (dinhtungdu) [reply .github/workflows/test.yml]: Thanks for the head up! Fixed it.

**Review by dkotter — DISMISSED** [review]

**Comment by jeffpaul** [comment]
@dkotter any chance you've got ideas on why the WP Acceptance test is failing?  @dinhtungdu and I couldn't get anywhere on that in reviewing that together earlier this week.

**Comment by jeffpaul** [comment]
Given that we're unlikely to resolve this issue before we'd want to release 1.6.2, I'm punting this to our next planned release in hopes of having things fully migrated off Travis and onto GitHub Actions by then.  (And yes, I realize this isn't really release-dependent, that's just how my brain works in this context.)

**Comment by jeffpaul** [comment]
@dinhtungdu maybe something's off in that Snapshot?

**Comment by jeffpaul** [comment]
@dinhtungdu what do you think about splitting this PR into two such that the part that has the linting working gets merged in while we work to get the testing action working?

**Comment by dinhtungdu** [comment]
@jeffpaul This PR is ready.

**Pull request merged**